src/components/ButtonDropdown.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Button } from "components/Button";
+import Icon from "@leafygreen-ui/icon";
+import Card from "@leafygreen-ui/card";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+
+interface Props {
+  disabled?: boolean;
+  setIsVisibleDropdown?: (v: boolean) => void;
+  loading?: boolean;
+  isVisibleDropdown?: boolean;
+  dropdownItems: JSX.Element[];
+  dataCyBtn?: string;
+  dataCyDropdown?: string;
+}
+
+export const ButtonDropdown: React.FC<Props> = ({
+  disabled = false,
+  loading = false,
+  setIsVisibleDropdown = function noop(v: boolean) {
```

Comment:
lets use arrow functions `() => undefined`. When the eslint rules are brought in i think this will be enforced more.